ui/redux/selectors.js

```diff
@@ -433,6 +433,7 @@ export const getRnaSeqSignificantJunctionData = createSelector(
           junctionLocus: `${row.chrom}:${row.start}-${row.end} ${row.strand}`,
           idField: getSpliceId(row),
           familyGuid: individualsByGuid[individualGuid].familyGuid,
+          individualName: individualsByGuid[individualGuid].displayName || individualGuid,
```

Comment:
what is the reason for this change?

Reply:
but why add the fallback value? If we are not showing the data results page theres no need to add this, it can just be null